raw_description/urdf/base_long/base.urdf.xacro

```diff
@@ -84,7 +86,7 @@
         <origin xyz="0 0 0.209" rpy="0 0 0" />
         <geometry>
           <mesh filename="package://raw_description/meshes/base/raw_base_long.stl"/>
-          <!--box size="1.04 0.58 0.31" /-->
+          <!--<box size="0.93 0.49 0.31" />-->
```

Comment:
why the changes in a commented line?

this review comment applies to all files in this pr

Reply:
no I mean, why the values have changed while still being commented?

Reply:
the commented box geometry was used because @ipa-srd-rd relied on the robot's collision mesh being detected in other robots scans
obviously, the current mesh does not have the plastic side panels modeled and the current box mesh is covers the field of view of the laser scanners

best solution would be to add the plastic side panels to the base.urdf.xacro (as new links)